modyn/supervisor/internal/evaluation_result_writer/__init__.py

```diff
@@ -6,7 +6,6 @@
 
 from .abstract_evaluation_result_writer import AbstractEvaluationResultWriter  # noqa: F401
 from .json_result_writer import JsonResultWriter  # noqa: F401
-from .log_result_writer import LogResultWriter  # noqa: F401
 from .tensorboard_result_writer import TensorboardResultWriter  # noqa: F401
```

Comment:
`LogResultWriter` is introduced in the [previous matrix eval PR](https://github.com/eth-easl/modyn/pull/334/files#diff-f444432c981ee0f1b1e27538f2aceb5d7ceb344df60c2791f8d118e17e7b7ee7) which is nothing different from `JsonResultWriter`.